.gitignore

```
@@ -1,11 +1 @@
-.Rproj.user
-.Rhistory
-.Rdata
-.httr-oauth
-.quarto
-*.Rproj
-
-experiments
-docs
-/doc/
-/Meta/
+*.html
```

Revert these changes :bow:

```suggestion
.Rproj.user
.Rhistory
.Rdata
.httr-oauth
.quarto
*.Rproj

experiments
docs
/doc/
/Meta/
*.html
```

---

I don't even know how that happened - sorry! Of course.